app/code/Magento/SalesGraphQl/Model/Formatter/Order.php

```diff
@@ -17,14 +20,19 @@
  */
 class Order
 {
+    private TimezoneInterface $timezone;
```

Comment:
Add doc-block as follows:

```suggestion
/**
     * @var TimezoneInterface $timezone
     */
    private TimezoneInterface $timezone;
```